packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.tsx

```diff
@@ -23,7 +24,24 @@ export class PaymentProvisioning {
   @State() errorMessage: string = '';
   @State() currentStep: number = 0;
   @State() totalSteps: number = 5;
-  @Event({eventName: 'click-event'}) clickEvent: EventEmitter<BusinessFormClickEvent>;
+  @Event({ eventName: 'click-event' }) clickEvent: EventEmitter<BusinessFormClickEvent>;
+
+  analtyics: JustifiAnalytics;
```

Comment:
`analytics` is misspelled in this file - but its correct in every other component file! 

Just need to update this from `analtyics` => `analytics`